src/models/NavigationBar.coffee

```diff
@@ -119,31 +123,30 @@ class NavigationBar
         btnParams = obj.toParams()
         if obj.imagePath?
           btnParams.imagePath = relativeTo + obj.imagePath
-
         return btnParams
 
       if typeof AndroidAPIBridge is 'undefined' # no AndroidAPIBridge on iOS
-        locations = ["right", "left"]
 
-        for location in locations
+        for location in ["right", "left"]
           steroids.debug "steroids.navigationBar.setButtons constructing location #{location}"
-          @buttonCallbacks[location] = []
           params[location] = []
 
           if options[location]?
             for button in options[location]
               buttonParameters = buttonParametersFrom(button)
-              callback = button.onTap ? ->
```

Comment:
can the onTap functionality be preserved on Steroids level too so existing code doesn't break? Or will we just keep compatibility on the Supersonic level and have the Steroids.js update be breaking?